src/sentry/static/sentry/app/views/projectsDashboard/deploys.jsx

```diff
@@ -63,14 +63,18 @@ class Deploy extends React.Component {
     return (
       <DeployRow justifyContent="space-between">
         <Environment>{deploy.environment}</Environment>
-        <Version>
-          <StyledLink
-            to={`/organizations/${organization.slug}/releases/${deploy.version}/?project=${project.id}`}
-          >
-            {deploy.version}
-          </StyledLink>
-        </Version>
-        <Flex width={90} justifyContent="flex-end">
+
+        <StyledTextOverflow>
+          <Version
+            version={deploy.version}
+            orgId={organization.slug}
+            projectId={project.id}
+            tooltipRawVersion
+            truncate
+          />
+        </StyledTextOverflow>
+
+        <Flex width={90} justifyContent="flex-end" flexGrow={1} flexShrink={0}>
```

Comment:
Shouldn't we avoid using `reflexbox`?

Reply:
We should, this was already there and the refactor of `reflexbox` seemed a bit out of the scope of this PR.